.travis.yml

```diff
@@ -1,4 +1,4 @@
-# This is required to get Travis to run Python 3.7
+# This is required to get Travis to run Python 3.7 and higher versions of Python
```

Comment:
Can you confirm this is still required?

Reply:
Oh, @inglesp, it's no longer required.

https://docs.travis-ci.com/user/languages/python/

Snapshots of that page obtained from The Way back Machine show that it was rectified/solved sometime between July 13, 2019 and August 29, 2019.

I'll be glad to take that out when I get to my computer.

Thank you for the catch.